docs/docs/setup-vast/install/linux.md

```diff
@@ -0,0 +1,77 @@
+# Linux
+
+Use our [pre-built build packages](/docs/setup-vast/download#packages) or [build
+from source](/docs/setup-vast/build) to install VAST on any Linux distribution.
+
+To deploy VAST as system service, you can use our [systemd
+configuration](#systemd).
+
+## systemd
+
+VAST has native [systemd
+support](https://github.com/tenzir/vast/tree/master/systemd/) for running as
+system service. The service is sandboxed and runs with limited privileges.
+
+### Prepare the host system
+
+Please note that all subsequent commands require `root` privileges. The service
+requires a user and group called `vast`. You can create them as follows.
+
+```bash
+useradd --system --user-group vast
+```
+
+Make sure that you don't grant any special rights to this user, e.g., do not
+enable `sudo` or other privileged commands. Once the user exists, you should
+create the directory for VAST's persistent data and change the permissions such
+that it is owned by the new `vast` user:
+
+```bash
+mkdir -p /var/lib/vast
+chown -R vast:vast /var/lib/vast
+```
+
+The systemd unit passes a
+[vast.yaml](https://github.com/tenzir/vast/tree/master/systemd/) configuration
+file to the VAST process. Make sure that the new user can read the `vast.yaml`.
+
+### Configure the unit
+
+Before you begin, find the line beginning with `ExecStart=` at the very bottom
+of the `[Service]` section in the unit file. Depending on your installation path
+you might need to change the location of the `vast` binary and configuration
+file.
+
+```config
+ExecStart=/path/to/vast --config=/path/to/vast.yaml start
```

Comment:
Why not just recommend using the default directory, e.g., `$PREFIX/etc/vast/vast.yaml`? Then it wouldn't need to be specified here. We also don't specify it in the example in the repository.

Reply:
I don't know. @lava or @tobim might know.

(The content is unadapted from the existing docs.)

Reply:
Note that duplicating the config from `$PREFIX/etc/vast/vast.yaml` with `--config` is not side-effect free per sc-32665; we should definitely not be recommending doing so, and if you want to see the used config files that already is available in `vast status | jq .system`.

Reply:
I removed the contested `--config` option. If there is a strong opinion to keep it, I would create another internal story.